### PR TITLE
Adding try/catch to ChangingMuteStatus and ChangingIntercomMuteStatus

### PR DESCRIPTION
**Pull request by Thundermaker300**

### CODE REVIEW

File: Exiled.Events/Patches/Events/Player/ChangingIntercomMuteStatus.cs [hunk 1]
```diff
@@ -22,25 +23,33 @@ internal static class ChangingIntercomMuteStatus
...
+
+                    return false;
                 }
 
-                return false;
+                return true;
+            }
+            catch (Exception e)
+            {
+                Exiled.API.Features.Log.Error($"Exiled.Events.Patches.Events.Player.ChangingIntercomMuteStatus: {e}\n{e.StackTrace}");
```
Comment (iRebbok) [inline Exiled.Events/Patches/Events/Player/ChangingIntercomMuteStatus.cs:50]:
```suggestion
                Exiled.API.Features.Log.Error($"{typeof(ChangingMuteStatus).FullName}.{nameof(Prefix)}:\n{e}");
```
And do the same with another patch

**Review by iRebbok — CHANGES_REQUESTED** [review]

File: Exiled.Events/Patches/Events/Player/ChangingIntercomMuteStatus.cs (hunk 1 above)
Comment (Thundermaker300) [reply Exiled.Events/Patches/Events/Player/ChangingIntercomMuteStatus.cs:50]: gotcha, copied this from a different patch

**Review by iRebbok — APPROVED** [review]
